doc/contributing/quick-start.chapter.md

```diff
@@ -50,28 +53,20 @@ To add a package to Nixpkgs:
 
    - A list of schemes for `mirror://` URLs can be found in [`pkgs/build-support/fetchurl/mirrors.nix`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/fetchurl/mirrors.nix).
 
-   The exact syntax and semantics of the Nix expression language, including the built-in function, are described in the Nix manual in the [chapter on writing Nix expressions](https://hydra.nixos.org/job/nix/trunk/tarball/latest/download-by-type/doc/manual/#chap-writing-nix-expressions).
-
-4. Add a call to the function defined in the previous step to [`pkgs/top-level/all-packages.nix`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/all-packages.nix) with some descriptive name for the variable, e.g. `libfoo`.
-
-   ```ShellSession
-   $ emacs pkgs/top-level/all-packages.nix
-   ```
-
-   The attributes in that file are sorted by category (like “Development / Libraries”) that more-or-less correspond to the directory structure of Nixpkgs, and then by attribute name.
+   To read more about the Nix language, see [Nix language basics](https://nix.dev/tutorials/first-steps/nix-language).
 
-5. To test whether the package builds, run the following command from the root of the nixpkgs source tree:
+4. To test whether the package builds, run the following command from the root of the nixpkgs source tree:
```

Comment:
I've learned that you can always use `1.` in markdown and it will count correctly for you in the html. Looks a bit weird in code review, but reduces diff noise.